### ISSUE TITLE
Directory based rules or alpine docker image

### ISSUE THREAD

**Issue by ferhatelmas** [bloom42/flint#9]
#3 already mentions directory based rules so I guess it's planned for improvement.

However, until that moment, we might go with our own version with some utilities in the image unless it was stratch.

What is your suggestion ?
* wait for improvement
* wait for new image based on alpine, for example
* roll our own image as our needs

**Comment by z0mbie42** [comment]
On the other hand I don't think  I'll change the `scratch` Docker image for another one for security reasons (to not increase the attack surface).

**Comment by ferhatelmas** [comment]
Proposed improvement answers what we want.

In the mono repo, we're applying conventions according to used languages where there are multiple of them as you can imagine. That's why we need to apply rules on the directories.

If we get it, there is no need to another docker image.

**Comment by z0mbie42** [comment]
Hi,
I apologize for the delay, I had a lot of work.

The feature discussed is available as of  2499fb3 (v2.2.0).

You can now filter directories / files like that:
```sane
{
    pattern = "*",
    type = "directory", # optional, default to "any", valid values are [file, directory, any]
    convention = "snake_case",
},
```
